src/test/Events/__snapshots__/EventTypes.test.tsx.snap

```diff
@@ -399,7 +399,7 @@ Array [
             aria-label="Details"
             aria-labelledby="simple-node0 expandable-event-type-row-00"
             className="pf-c-button pf-m-plain"
-            data-ouia-component-id="OUIA-Generated-Button-plain-1"
+            data-ouia-component-id="OUIA-Generated-Button-plain-5"
```

Comment:
Hmm pretty odd that this id is once again updated.

Reply:
Yeah, I'm not sure what's going on with that.

Reply:
seems like something is buggy with the latest PR react-table version.